include/CLI/FormatterFwd.hpp

```diff
@@ -57,6 +57,7 @@ class FormatterBase {
     FormatterBase() = default;
     FormatterBase(const FormatterBase &) = default;
     FormatterBase(FormatterBase &&) = default;
+    FormatterBase &operator=(const FormatterBase &) = default;
```

Comment:
What about the move assignment operator?

Reply:
The rule of 5 states you should have none or all of these. Guessing the tool hasn't been updated for moves, perhaps? (It seems to be applying the rule of 3).

Reply:
```suggestion
    FormatterBase &operator=(const FormatterBase &) = default;
    FormatterBase &operator=(FormatterBase &&) = default;
```